3-UsingAPIs/Week2/homework/ex5-vscDebug.js

```diff
@@ -18,14 +18,15 @@ function renderLaureate({ knownName, birth, death }) {
 }
 
 function renderLaureates(laureates) {
-  laureates.forEach(renderLaureate);
+  laureates.forEach((each) => renderLaureate(each));
```

Comment:
`each` is a bit of a vague name, I would choose something like "laureate" because that is what you are actually looping over 🙂 

```suggestion
  laureates.forEach((laureate) => renderLaureate(laureate));
```